docs/admin/troubleshooting/jcmd/docker.rst

```diff
@@ -32,57 +32,18 @@ Run ``jcmd`` inside container
 The commands below use ``docker``. In the same spirit, you can also use
```

Comment:
`podman` has no `--user` so looks like this no longer applies?

Reply:
Bummer. Dito: I will use `su` instead of proprietary options.

Reply:
It looks like both `podman run` and `podman exec` also accept the `--user` option?

-- https://docs.podman.io/en/stable/markdown/podman-run.1.html#user-u-user-group
-- https://docs.podman.io/en/stable/markdown/podman-exec.1.html#user-u-user-group